scripts/run-ci-build.sh

```diff
@@ -1,20 +1,14 @@
 #!/bin/bash
 
+# TODO -euo pipefail isn't optimal but I haven't got the energy to improve further -Anatol
 set -euxo pipefail
 
+# the old filter removed crowtty and manganese, but manganese isn't in deps anymore
+# TODO crowtty is currently being filtered because of netlify; when we migrate off of them
+# the del part can be done away with.
 defaultmembers=$( \
     cargo metadata --format-version 1 | \
-    jq .workspace_default_members | \
-    grep -E '  ".*' | \
-    # crowtty's dependencies can't easily be installed on netlify
-    grep -v 'crowtty' | \
-    # manganese depends on `libudev` (transitive dep via one of its' bindeps)
-    # which can't be installed on CI.
-    grep -v 'manganese' | \
-    cut -d" " -f3 | \
-    cut -d'"' -f2 | \
-    sed -E 's/(.*)/-p \1 /g' | \
-    tr -d '\n' \
+    jq -r '.workspace_default_members | del(.[] | select(contains("crowtty"))) | to_entries[] |"-p \(.value)"'
```

Comment:
hmm, this can't easily be made multi-line, i assume?

Reply:
pushed a fix that seems to work